docs/CHANGELOG.rst

```diff
@@ -44,17 +50,10 @@ Fixed
   process is finished
 - Remove unrelated permissions occasionally listed among group
   permissions on ``permissions`` endpoint
-- Fix ``ResolwePermissionsMixin`` to work correctly with multi-words 
-  model names, i.e. DescriptorSchema
-
-
-==================
-3.0.1 - 2017-09-15
```

Comment:
Why was this deleted? You should also include change logs for all `3.x` versions.

Reply:
I've reverted the change and added information about all `3.x` releases that happened in the mean time.

Reply:
Thanks!